source/mas_piloot_steekproefdesign/05_analyse_gegevens.Rmd

```diff
@@ -1097,29 +1271,169 @@ include_graphics(file.path("media", "distance", "per_stratum",
 
-### Toestand beheerovereenkomsten {#beheerovereenkomsten}
+```{r densiteitskaart, out.width = "100%", fig.cap = "Aantal broedparen Veldleeuwerik per 100 ha per rastercel voor het steekproefkader van de Oostelijke leemstreek met contourlijnen. Posterior mediaan als maat voor het centrale tendens. Q2.5 is de ondergrens van het 95 \\% geloofwaardigheidsinterval en Q97.5 de bovengrens."}
+include_graphics(file.path("media", "spatiale_densiteiten",
+                           "veldleeuwerik_dens_ol2.pdf"))
+```
+
+```{r relfout-densiteitskaart, out.width = "100%", fig.cap = "Spatiale distributie van de relatieve foutenmarge op het aantal broedparen Veldleeuwerik per 100 ha per rastercel."}
+include_graphics(file.path("media", "spatiale_densiteiten",
+                           "veldleeuwerik_dens_ol3.pdf"))
+```
+
+#### Discussie densiteitskaart
+
+We gebruikten een tweefasenaanpak om een model te creëren waarmee we de dichtheidsschattingen konden extrapoleren naar een groter studiegebied, nl.
+het steekproefkader in de Oostelijke leemstreek.
+De eerste fase is het schatten van de detectiekans door het fitten van een detectiecurve (Sectie \@ref(res-distance-stratum)).
+De tweede fase is de ruimtelijke modellering via een soepele *smoothing function*.
+Tweefasemodellen hebben het nadeel dat de onzekerheid van de detectiefunctie en de ruimtelijke modellen op de juiste manier gecombineerd moeten worden [@miller2013].
+Wij hebben hier de onzekerheid van het detectieproces genegeerd wat te smalle geloofwaardigheidsintervallen kan opleveren voor schattingen van de densiteit.
+Bovendien zijn we voorlopig niet dieper ingegaan op de keuze voor de smoothing function $f()$.
+Ook hier kunnen verschillende afwegingen gemaakt worden [@wood2017].
+In dit stadium van de pilootstudie was het niet mogelijk dieper in te gaan op deze tekortkomingen.
+Bij de uitbreiding van een MAS voor heel Vlaanderen, zal deze methode verder op punt gesteld worden.
 
-#### Inleiding
+Indien we subgebieden willen afbakenen, bijvoorbeeld om interessante kernzones te identificeren, kunnen we gridcellen clusteren en daarna de predictie van de cluster vergelijken met een vooraf afgesproken waarde (bv., Is het aantal broedparen per 100 ha significant meer dan 10?).
+Hierbij moeten we een een methode vinden waarbij we op correcte wijze de onzekerheid berekenen op clusterniveau [@wadoux2023].
+Een eerste mogelijkheid is om een variogram op te stellen o.b.v. de fitted values om een idee te krijgen van spatiale autocorrelatie.
+Deze info kan dan gebruikt worden om de onzekerheid op clusterniveau te bepalen.
+Een tweede mogelijkheid is om andere methodes en R packages te bekijken die dergelijke berekeningen integraal uitvoeren.
+Ook deze mogelijkheden zullen in een later stadium onderzocht moeten worden.
 
-\textcolor{red}{tekst}
+### Correlaties met verklarende variabelen {#beheerovereenkomsten}
```

Comment:
Ik zou in heel deze sectie niet spreken over correlaties. Correlaties zijn relaties tussen twee variabelen, terwijl we hier regressiemodellen fitten met 1 afh var en meerdere onafh vars. De titel kan misschien iets als "modellering in functie van verklarende variabelen" worden.